pages/database-management/monitoring.mdx

```diff
@@ -7,50 +7,102 @@ import { Callout } from 'nextra/components'
 
 # Monitoring
 
-In production systems, monitoring of applications is crucial, and that includes
-databases as well. Memgraph allows monitoring the database via WebSocket server and a
-HTTP server. 
+Monitoring of applications and databases is essential to be able to track the
+load, resources, logs, transactions, latency etc. 
 
-## Monitoring via WebSocket server
+Memgraph currently supports:
+
+- Logs tracking via WebSocket server 
+- Monitoring via HTTP server (Enterprise Edition)
+
+Log tracking is helpful for debugging purposes and monitoring the database
+operations.  
+ 
+In the Enterprise Edition, on top of log tracking,  Memgraph allows tracking
+information about transactions, query latencies, snapshot recovery latencies,
+triggers, bolt messages, indexes, etc., using an HTTP server.
```

Comment:
Just a suggestion for a bit clearer structure:
```suggestion
Memgraph currently supports:
1. **Log tracking via WebSocket server**: Log tracking is helpful for debugging purposes and monitoring the database operations. 
2. **Monitoring via HTTP server (Enterprise Edition)**: In the Enterprise edition, besides log tracking, Memgraph allows tracking information about transactions, query latencies, snapshot recovery latencies, triggers, bolt messages, indexes, and more, using an HTTP server.
```

Reply:
I agree, wanted to suggest the same

Reply:
Agree!

Reply:
Please avoid using terms such as "etc" and "and more". Prefer being thorough in listing everything the user might need or consider.